Mathlib/Computability/Language.lean

```diff
@@ -313,18 +313,16 @@ lemma reverse_zero : (0 : Language α).reverse = 0 := rfl
 lemma reverse_one : (1 : Language α).reverse = 1 := by
   simp [reverse, ← one_def]
 
-variable (l m)
-
```

Comment:
I'm happy with this change, but just because the commit message leaves room for doubt: this make no difference to whether the arguments are explicit, and is solely a stylistic change.

Reply:
Absolutely. I just think most of us agree that `variable` should not be used for explicit arguments.